receipt_dynamo/fix_exceptions.py

```
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+"""
+Script to systematically fix exception inconsistencies in receipt_dynamo data files.
+"""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)